sarc/cli/db/prometheus/backup.py

```diff
@@ -0,0 +1,220 @@
+import json
+import logging
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+from pathlib import Path
+from typing import Any
+
+from pydantic import BaseModel
+from tqdm import tqdm
+
+from sarc.client.job import _jobs_collection, JobStatistics, SlurmJob
+from sarc.config import config, TZLOCAL
+from sarc.jobs.sacct import update_allocated_gpu_type_from_nodes
+from sarc.jobs.series import (
+    _get_job_time_series_data_cache_key,
+    JOB_STATISTICS_METRIC_NAMES,
+)
+
+logger = logging.getLogger(__name__)
+
+
+DATE_FORMAT_HOUR = "%Y-%m-%dT%H:%M"
+
+
+class JobPrometheusData(BaseModel):
+    """Helper class to store Prometheus metrics for a job"""
+
+    cluster_name: str
+    job_id: int
+    submit_time: datetime
```

Comment:
This should use datetime_utc (from sarc.core.models.validators) as the type to ensure that the data put here will be in UTC timezone.

Reply:
Actually, when reading `submit_time` (and other dates) from DB, current code interprets it as UTC (as we expect), but immediately converts it to MTL:

https://mila-iqia.atlassian.net/browse/SARC-467?focusedCommentId=137194

```python
    @field_validator(
        "submit_time",
        "start_time",
        "end_time",
        "latest_scraped_start",
        "latest_scraped_end",
    )
    @classmethod
    def _ensure_timezone(cls, v: datetime | None) -> datetime | None:
        # We'll store in MTL timezone because why not
        return v and v.replace(tzinfo=UTC).astimezone(MTL)
```

So, `submit_time` is already in MTL timezone when I read it from SlurmJob, so typing with `datetime_utc` fails.

Reply:
That code should also be fixed, but it can wait for a further PR.  For now you can convert to UTC with `.astimezone(UTC)`.